tfe/resource_tfe_sentinel_policy.go

```diff
@@ -0,0 +1,167 @@
+package tfe
+
+import (
+	"fmt"
+	"log"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceTFESentinelPolicy() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceTFESentinelPolicyCreate,
+		Read:   resourceTFESentinelPolicyRead,
+		Update: resourceTFESentinelPolicyUpdate,
+		Delete: resourceTFESentinelPolicyDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"organization": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"policy": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"enforce_mode": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: func(v interface{}, k string) ([]string, []error) {
+					switch v.(string) {
+					case "advisory", "hard-mandatory", "soft-mandatory":
```

Comment:
(as above) we can use `validation.StringInSlice([]string{"foo","bar"}, false)` here

Reply:
if there's enum's we can also do this with `string(tfe.EnforcementModeFoo)`

Reply:
Nice... Fixed in the next commit.